storage/libradb/src/lib.rs

```diff
@@ -404,6 +404,14 @@ impl LibraDB {
         Ok(())
     }
 
+    pub fn get_state_restore_receiver(
```

Comment:
not sure if we should call this ***handler like the back up handler

Reply:
Great idea. Will add that layer when it handles more than state (i.e. Transactions / ledger_infos, etc)